index.dd

```diff
@@ -53,6 +53,28 @@ void main()
 )
 )
 $(EXTRA_EXAMPLE
+----
+void main()
+{
+    // Sort inplace across all arrays
+    // without any memory allocation
+    import std.stdio : writefln;
+    import std.algorithm.sorting : sort;
+    import std.range : chain;
+
+    int[] arr1 = [4, 9, 7];
+    int[] arr2 = [5, 2, 1, 10];
+    int[] arr3 = [6, 8, 3];
+    // @nogc functions are guaranteed by the compiler
+    // to be without any GC allocation
+    () @nogc {
```

Comment:
Yep, of course, but this depends on `writefln` (or similar) to be `@nogc` ...
And the [`assumeNoGC`](https://p0nce.github.io/d-idioms/#Bypassing-@nogc) hack seemed to ugly for a frontpage example.